docs/api.md

```diff
@@ -759,6 +759,9 @@ _Type:_ `object`
 
 This lists all options for [`execa()`](#execafile-arguments-options) and the [other methods](#methods).
 
+> [!IMPORTANT] Differences from `child_process.spawn` API
+> - `stdio: [..., 'ipc']` is deprecated and will produce a runtime error in version 10
```

Comment:
Could you please remove both `>` and `[!IMPORTANT]`/`[!NOTE]`? It's not a bad thing, but it's not consistent with the rest of the documentation, which does not use this style. Thanks!